fetch.bs

```diff
@@ -7800,12 +7831,10 @@ method steps are:
  <li><p>Return <var>p</var>.
 </ol>
 
-<p>To <dfn>abort fetch</dfn> with a <var>promise</var>, <var>request</var>, and
-<var>responseObject</var>, run these steps:
+<p>To <dfn lt="abort the fetch() call" export id=abort-fetch-algorithm>abort a fetch() call</dfn> with a
```

Comment:
We should keep the existing ID `abort-fetch` here. Also, if we rename this to include the name of the method, `fetch()` should be inside `<code>`.